app/population/middleware/index.js

```diff
@@ -1,7 +1,9 @@
 const redirectBaseUrl = require('./redirect-base-url')
+const setPopulationId = require('./set-population')
```

Comment:
Should this be `setPopulation` instead of `setPopulationId`?

Reply:
Renamed